configdata.pm.in

```diff
@@ -15,21 +15,6 @@
      # Make the path absolute at all times
      $path = abs_path($path);
```

Comment:
Can this be simplified to `return abs_path($path);`? Can we drop `_fixup_path` entirely and inline `abs_path()` in the two callers directly?